.github/CODEOWNERS

```diff
@@ -87,3 +87,10 @@
 
 #SIC SEMPER TYRANNIS
 /code/modules/hydroponics/grown/citrus.dm @optimumtact
+
+# Fulp Files
+
+/fulp_modules/main_features/bloodsuckers/ @JohnFulpWillard
+/fulp_modules/main_features/jobs/ @JohnFulpWillard @SgtHunk
+/fulp_modules/main_features/mentorhelp/ @JohnFulpWillard
```

Comment:
'Scuse me, but I am also the owner of the mentor code since I was the one who fixed it.

- Sincerely, EgorDinamit.




Now, speaking seriously, you probably want to remove mentions of the /tg/ coders and non-maintainers of this repo.
All this file does (in theory) is ask for a review each time someone edits files in these directories, and I've been told it doesn't ask for a review if you have literally 0 permissions on the repository.